src/components/marsPhotoViewer/ImageSelector.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect, useState } from "react";
+import { getImagesFromApi, imageApiModel } from "../../api/imagesApi";
+import "./ImageSelector.scss";
+
+export function ImageSelector(props: {
+    setState: React.Dispatch<React.SetStateAction<string>>,
+    date: string,
+    rover: string,
+    camera: string,
+}) {
+    const [imgUrl, setImgUrl] = useState<imageApiModel[]>();
+
+    useEffect(() => {
+        getImagesFromApi(props.date, props.rover, props.camera)
+            .then(data => setImgUrl(data))
+            .then(() => { if (imgUrl) props.setState(imgUrl[0].img_src) });
+    }, []);
+
+    const urls = imgUrl ? imgUrl.slice(0, Math.min(imgUrl.length, 12)).map(url => <img src={url.img_src} key={url.id} className="individualImage" alt='Mars image'
+        onClick={() => props.setState(url.img_src)} />) : <p>No Image to Display</p>;
```

Comment:
Can you reformat this to make it easier to read?

```
const urls = imgUrl 
    ? imgUrl
        .slice(0, Math.min(imgUrl.length, 12))
        .map(url => {
            <img src={url.img_src} 
                key={url.id} 
                className="individualImage" 
                alt='Mars image'
                onClick={() => props.setState(url.img_src)} />
        }) 
    : <p>No Image to Display</p>;
```

Reply:
And can we rename this to `images` as we are actually returning an image element